pkg/cli/gh_gc_issues.go

```diff
@@ -0,0 +1,87 @@
+package cli
+
+import (
+	"context"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v55/github"
+
+	http2 "github.com/wolfi-dev/wolfictl/pkg/http"
+	"golang.org/x/oauth2"
+	"golang.org/x/time/rate"
+
+	"github.com/spf13/cobra"
+	"github.com/wolfi-dev/wolfictl/pkg/gh"
+	wgit "github.com/wolfi-dev/wolfictl/pkg/git"
+)
+
+func Issues() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "issues",
+		DisableAutoGenTag: true,
+		SilenceUsage:      true,
+		Short:             "Issues garbage collection commands used with GitHub",
+		Long: `Issues garbage collection commands used with GitHub
+
+Examples:
+
+wolfictl gc issues https://github.com/wolfi-dev/versions --match "version-stream:"
+`,
+		Args: cobra.RangeArgs(1, 1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ts := oauth2.StaticTokenSource(
+				&oauth2.Token{AccessToken: os.Getenv("GITHUB_TOKEN")},
+			)
+			client := &http2.RLHTTPClient{
+				Client: oauth2.NewClient(context.Background(), ts),
+
+				// 1 request every (n) second(s) to avoid DOS'ing server. https://docs.github.com/en/rest/guides/best-practices-for-integrators?apiVersion=2022-11-28#dealing-with-secondary-rate-limits
+				Ratelimiter: rate.NewLimiter(rate.Every(5*time.Second), 1),
+			}
+
+			return gcIssues(client, args[0], match)
+		},
+	}
+
+	cmd.Flags().StringVar(&match, "match", "", "pattern to match issues against")
+
+	return cmd
+}
+
+func gcIssues(ghclient *http2.RLHTTPClient, repo, match string) error {
```

Comment:
question: does it make any sense to verify that `match` is non-empty before using it in `HasPrefix`, just to avoid mass closing of issues?

Reply:
SGTM